### DISCUSSION TITLE
Biased questions and biased presentation of results

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#32] [Past Thinking Caps]
The phrasing of questions can influence how people choose to respond, which can skew the results. A silly example is this week’s discussion question: “What are problems with the way we use data and GIS?” I was initially really stuck on how to respond because it read to me as an objective question, and I didn’t want to just echo what we covered in class. I found some more serious examples at [11 Biased & Unbiased Question Examples in Surveys (Formplus)](https://www.formpl.us/blog/biased-survey-question-example), a lot of which I didn’t initially realize were biased!

In my INF STD 20 class, we read [Machine Bias (ProPublica, 2016)](https://www.propublica.org/article/machine-bias-risk-assessments-in-criminal-sentencing), which concluded that risk-assessment software is biased against Black people. This reminded me of last year’s Proposition 25, which proposed replacing cash bail because it disproportionately impacts people of color and low-income people with risk assessments, which also disproportionately impact people of color and low-income people. From what I remember, Pro-Prop 25 people framed risk assessments as being better than cash bail by just not saying anything about issues associated with risk assessments. The omission of that information must have resulted in people voting yes on Prop 25 when they wouldn’t have if they understood its full implications.

(I haven’t read [The Age of Secrecy and Unfairness in Recidivism Prediction (HDSR, 2020)](https://hdsr.mitpress.mit.edu/pub/7z10o269/release/4) yet, but in its abstract, it addresses issues with the ProPublica longform that were a result of lack of transparency with the model they were studying. Also relevant to the discussion question haha.)

**Comment by Wny-Duong** [comment]
I wanted to add onto the topic of risk assessment you brought up with the Machine Bias article. There's actually a very interesting phenomenon called proxy discrimination, where certain variables are used as a proxies for other variables that would be discriminatory against protected groups, that comes up when it comes to machine-learning assisted software dealing with risks. It's briefly brought up in the article you brought up as well.

> "It is difficult to construct a score that doesn’t include items that can be correlated with race — such as poverty, joblessness and social marginalization. “

Historically in the US, this was something that's been understood in the form of redlining. In recent times however, it's something that emerges when it comes to acquiring health insurance, fair interest rates on loans, among other things. When things such as race, sex, and other labels are prohibited from being used to evaluate people for such services, it's things like zip codes, credit scores, hair product purchases, and video streaming history that are utilized as a proxy for those things ([There's a massive legal article just about this topic published if you want more in-depth detail.](https://ilr.law.uiowa.edu/print/volume-105-issue-3/proxy-discrimination-in-the-age-of-artificial-intelligence-and-big-data/).  Stuff like this normally happens intentionally, but with the increasing digitization of things and rise of big data in modern business and life, machine learning algorithms have a tendency to do this unintentionally as well.

**Comment by brillianz** [comment]
This reminds me of work that's been disseminated by a professor at UCLA on predictive policing. Jeffery Brantingham, the anthropology professor associated with this research, has developed several studies advocating for this work. The irony, of course, is that these algorithms have been imposed most directly on communities in South Central Los Angeles. There has been a back-and-forth dialogue across both sides of discourse relating to racism, [with a 2018 study the most recent subject of this response](https://www.math.ucla.edu/~bertozzi/papers/BrantinghamMohlerA.pdf). 
I am also familiar with ongoing dialogues surrounding facial recognition software. [This interview](https://www.democracynow.org/2020/6/26/racist_facial_recognition_technology_joy_buolamwini) on Democracy Now features Joy Buolamwini, a researcher at the MIT Media Lab and founder of the Algorithmic Justice League and her thoughts on this emerging software. It is being use more and more in countries like China, and her contributions allude to different uses according to the community it is placed in. The Algorithmic Justice League above may be something that interests you~
